packages/gallery/src/components/item/itemHover.js

```diff
@@ -54,22 +54,109 @@ export default class ItemHover extends GalleryComponent {
     return true;
   }
 
+  getOverlayStyle() {
+    const { styleParams, imageDimensions } = this.props;
+    let {
+      overlayPosition,
+      overlaySize: requiredOverlaySize,
+      overlaySizeType,
+      overlayPadding,
+    } = styleParams;
+    let { width, height } = imageDimensions;
+    let style = {};
+    let overlayHeight, overlayWidth, overlaySize;
```

Comment:
can you try and make the entire method with `const` only? I want us to stop adding mutations to the code